src/views/Home/View.tsx

```diff
@@ -22,6 +24,16 @@ const View: React.FC<NextPage> = () => {
    * be added in SALEOR-1566.
    */
   React.useEffect(() => {
+    const getToken = async () => {
+      const claims = await getIdTokenClaims();
+      const idToken = claims && claims["__raw"];
```

Comment:
what's this?

Reply:
I tried to output `id_token`, but I think `getAccessTokenSilently` will be enough.

https://github.com/auth0/auth0-react#call-an-api